packages/web/test/src/lib/myalgowallet-mode.ts

```diff
@@ -0,0 +1,60 @@
+import MyAlgoConnect from "@randlabs/myalgo-connect";
+import algosdk, { Account } from "algosdk";
+import assert from "assert";
+
+import { MyAlgoWalletSession, types } from "../../../src";
+import { algoexplorerAlgod, mkTxParams } from "../../../src/lib/api";
+import { mkTransaction } from "../../../src/lib/txn";
+import { HttpNetworkConfig } from "../../../src/types";
+import MyAlgoConnectMock from "../../mocks/myalgowallet-mock";
+
+describe("Webmode - MyAlgo Wallet test cases ", () => {
+	let connector: MyAlgoWalletSession;
+	let sender: Account;
+	let receiver: Account;
+	let algodClient: algosdk.Algodv2;
+	const walletURL: HttpNetworkConfig = {
+		token: "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+		server: "http://localhost",
+		port: 4001,
+	}
+	algodClient = algoexplorerAlgod(walletURL);
+
+	beforeEach(() => {
+		MyAlgoConnect
+		sender = algosdk.generateAccount();
+		receiver = algosdk.generateAccount();
+		connector = new MyAlgoWalletSession(walletURL, new MyAlgoConnectMock());
+
+	});
+
+	it("Should executeTx without throwing an error", async () => {
```

Comment:
Don't use the arrow method. Use normal functions.
Reason: https://mochajs.org/#arrow-functions

Reply:
but I m not using any context in any of the cases, so it shouldn't be a problem

Reply:
Yes, but we have to use convert it to normal methods. It's needed.